gba/data/wiki_dataset.py

```diff
@@ -0,0 +1,97 @@
+from collections import defaultdict
```

Comment:
Move this file to content to gba/tools/search/data.py (and merge the README files)?